packages/@aws-cdk/aws-ecs/test/ec2/integ.spot-drain.ts

```diff
@@ -0,0 +1,44 @@
+import ec2 = require('@aws-cdk/aws-ec2');
```

Comment:
Thanks for adding the integ test. It would be great if you can also add a unit test for the new property.

Reply:
fixed https://github.com/aws/aws-cdk/pull/4360/commits/45bef1da7735bc55ff50668e70c7c3c9349b6944